pyproject.toml

```diff
@@ -20,6 +20,10 @@ newhelm_together = {version = "*", optional = true}
 newhelm_huggingface = {version = "*", optional = true}
 perspective_api = {version = "*", optional = true}
 tomli = "^2.0.1"
+scikit-learn = "^1.4.1.post1"
```

Comment:
These dependencies should be on `plugins/standard_tests/`, not the overall one, as we only need them for the plugin.

Reply:
Fixed!